packages/0x.js/src/order_watcher/order_state_watcher.ts

```diff
@@ -133,8 +133,12 @@ export class OrderStateWatcher {
         delete this._orderStateByOrderHashCache[orderHash];
         const exchange = (this._orderFilledCancelledLazyStore as any)._exchange as ExchangeWrapper;
         const zrxTokenAddress = exchange.getZRXTokenAddress();
+        
         this._removeFromDependentOrderHashes(signedOrder.maker, zrxTokenAddress, orderHash);
-        this._removeFromDependentOrderHashes(signedOrder.maker, signedOrder.makerTokenAddress, orderHash);
+        if (zrxTokenAddress !== signedOrder.makerTokenAddress) {
+            this.removeFromDependentOrderHashes(signedOrder.maker, signedOrder.makerTokenAddress, orderHash);
```

Comment:
theres an underscore missing here

Reply:
How did this compile 😕

Reply:
It didn't. But I changed that before publishing